ui/Screen/Lock.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import * as session from "../src/session";
+
+  import { Button, Input } from "../DesignSystem/Primitive";
+
+  let passphrase = "";
+
+  const unlock = () => {
+    session.unseal(passphrase);
+  };
+
+  const onKeydown = (event: KeyboardEvent) => {
+    if (event.key === "Enter" && passphrase.length > 0) {
+      unlock();
+    }
+  };
+</script>
+
+<style>
+  .lock {
+    height: 100vh;
+    width: 100vw;
+    position: fixed;
+    z-index: 100;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+  }
+</style>
+
+<div class="lock">
+  <Input.Text
```

Comment:
This should be `Input.Password`.

<img width="370" alt="Screenshot 2020-10-29 at 10 05 13" src="https://user-images.githubusercontent.com/158411/97551420-3f8e0d00-19d3-11eb-8264-d7ba610b7877.png">

vs

<img width="330" alt="Screenshot 2020-10-29 at 10 42 02" src="https://user-images.githubusercontent.com/158411/97551550-677d7080-19d3-11eb-9d81-6a4408f124d2.png">

Reply:
Addressed in 51c9c77

Reply:
The <kbd>enter</kbd> key doesn't work for submission anymore.